main.py

```diff
@@ -1,13 +1,19 @@
 """
 Entry point for LTE Communication.
 """
+import pathlib
 import time
 
+from modules.common.camera.modules.camera_device import CameraDevice
 from modules.common.mavlink.modules import flight_controller
 
 
 MISSION_PLANNER_ADDRESS = "tcp:127.0.0.1:14550"
-DELAY_TIME = 1.0  # seconds
+DELAY_TIME_1 = 1.0  # seconds
```

Comment:
More descriptive name. The 1st loop is for waiting, the 2nd loop is for collecting and sending data.